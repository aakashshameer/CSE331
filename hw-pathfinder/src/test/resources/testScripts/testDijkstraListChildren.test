#List the children of these nodes numerically

CreateGraph childrenGraph

AddNode childrenGraph A
AddNode childrenGraph B
AddNode childrenGraph C

AddEdge childrenGraph A B 3.0
AddEdge childrenGraph B C 10.0
AddEdge childrenGraph A C 5.0
AddEdge childrenGraph B A 2.0

ListChildren childrenGraph A
ListChildren childrenGraph B

