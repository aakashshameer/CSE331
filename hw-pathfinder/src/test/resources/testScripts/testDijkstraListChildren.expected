#List the children of these nodes numerically

created graph childrenGraph

added node A to childrenGraph
added node B to childrenGraph
added node C to childrenGraph

added edge 3.000 from A to B in childrenGraph
added edge 10.000 from B to C in childrenGraph
added edge 5.000 from A to C in childrenGraph
added edge 2.000 from B to A in childrenGraph

the children of A in childrenGraph are: B(3.000) C(5.000)
the children of B in childrenGraph are: A(2.000) C(10.000)